core/src/main/java/org/apache/spark/api/java/function/package.scala

```diff
@@ -22,4 +22,4 @@ package org.apache.spark.api.java
  * these interfaces to pass functions to various Java API methods for Spark. Please visit Spark's
  * Java programming guide for more details.
  */
-package object function 
+package object function
```

Comment:
Yes I'm pretty sure you are wrong. See https://github.com/apache/spark/pull/13423/files#diff-c8ebb678d9e773dd03e05b0bca473d17R25 for an example of a file lacking a new blank line.